### DISCUSSION TITLE
Welcome to Nekos-API Discussions!

### DISCUSSION THREAD

**Discussion by Nekidev** [Nekos-API/Nekos-API#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dinaolmelak** [comment]
👍  A pleasure to get to connect with other members. I just recently came across the api. It is a really nice! 😄

**Comment by nevil-ing** [comment]
hello, am currently using the API for a wallpaper app. The previous version of the API had categories option. but on this newer version [3.0.0] i realised you have removed the category options plus some other features. Am using flutter for my project.
overrall i like the API

**Reply by Nekidev** [reply]
Hi! Categories weren't removed, they were renamed to `tags` (https://nekosapi.com/docs/changelog).

Which features were removed that you'd like to see in this version?

**Reply by nevil-ing** [reply]
oh thanks! i had  not processed that .
some features like user authentication, users uploading their own images.
But so far i like the new update and i have found what i needed most.

**Reply by Nekidev** [reply]
Yes, all those features were removed because that wasn't the original idea of the API, plus they worked terribly bad and there were lots of security issues. As I had to rewrite the API to move it to a shared server with NekoSauce (a more powerful VPS) I ended removing all those features. I'll eventually reimplement them in nekos.land, but as a feature of nekos.land instead of nekos API.

**Reply by nevil-ing** [reply]
oh okay then!

**Comment by Foster-11** [comment]
Why an Public API has Sex*al images?
Some Id's like 10 are dissgustings

**Reply by Nekidev** [reply]
You can always filter by image rating. Read the documentation for more information, as it's properly described there. https://nekosapi.com/
